morden/sass/_extra-child-theme.scss

```diff
@@ -247,6 +247,24 @@ body {
 /**
  * Blocks
  */
+
+// Constrain left-aligned blocks
+@media only screen and (min-width: 1280px) {
+	.entry-content > .wp-block-button:not(.alignleft):not(.alignright),
+	.entry-content > .alignleft {
+		margin-left: calc( 0.5 * (100% - calc( 782px - 32px)));
+		max-width: calc( 0.5 * calc( 782px - 32px));
+	}
+}
+
+// Constrain right-aligned blocks
+@media only screen and (min-width: 1280px) {
+	.entry-content > .alignright {
+		margin-right: calc( 0.5 * (100% - calc( 782px - 32px)));
+		max-width: calc( 0.5 * calc( 782px - 32px));
+	}
+}
+
```

Comment:
I forgot to see how this affected the other break points. Sorry about that. 

The following should fix the issue for all screen sizes. It’s a pretty big chunk of code but it fixes the issue properly. Make that change and I’ll go ahead and get this merged. 

```suggestion

// Add space for floated content
@media only screen and (min-width: 560px) {
	.entry-content > .wp-block-button:not(.alignleft):not(.alignright),
	.entry-content > .alignleft {
		margin-right: #{map-deep-get($config-global, "spacing", "horizontal")};
	}

	.entry-content > .alignright {
		margin-left: #{map-deep-get($config-global, "spacing", "horizontal")};
	}
}

// Constrain left-aligned blocks
@media only screen and (min-width: 640px) {
	.entry-content > .wp-block-button:not(.alignleft):not(.alignright),
	.entry-content > .alignleft {
		margin-left: calc( 0.5 * (100% - calc( 640px - 32px)));
		max-width: calc( 0.5 * calc( 782px - 32px));
	}
}

// Constrain right-aligned blocks
@media only screen and (min-width: 640px) {
	.entry-content > .alignright {
		margin-right: calc( 0.5 * (100% - calc( 640px - 32px)));
		max-width: calc( 0.5 * calc( 640px - 32px));
	}
}

// Constrain left-aligned blocks
@media only screen and (min-width: 782px) {
	.entry-content > .wp-block-button:not(.alignleft):not(.alignright),
	.entry-content > .alignleft {
		margin-left: calc( 0.5 * (100% - calc( 782px - 32px)));
		max-width: calc( 0.5 * calc( 782px - 32px));
	}
}

// Constrain right-aligned blocks
@media only screen and (min-width: 782px) {
	.entry-content > .alignright {
		margin-right: calc( 0.5 * (100% - calc( 782px - 32px)));
		max-width: calc( 0.5 * calc( 782px - 32px));
	}
}

// Constrain left-aligned blocks
@media only screen and (min-width: 1024px) {
	.entry-content > .wp-block-button:not(.alignleft):not(.alignright),
	.entry-content > .alignleft {
		margin-left: calc( 0.5 * (100% - calc( 782px - 32px)));
		max-width: calc( 0.5 * calc( 782px - 32px));
	}
}

// Constrain right-aligned blocks
@media only screen and (min-width: 1024px) {
	.entry-content > .alignright {
		margin-right: calc( 0.5 * (100% - calc( 782px - 32px)));
		max-width: calc( 0.5 * calc( 782px - 32px));
	}
}

// Constrain left-aligned blocks
@media only screen and (min-width: 1280px) {
	.entry-content > .wp-block-button:not(.alignleft):not(.alignright),
	.entry-content > .alignleft {
		margin-left: calc( 0.5 * (100% - calc( 782px - 32px)));
		max-width: calc( 0.5 * calc( 782px - 32px));
	}
}

// Constrain right-aligned blocks
@media only screen and (min-width: 1280px) {
	.entry-content > .alignright {
		margin-right: calc( 0.5 * (100% - calc( 782px - 32px)));
		max-width: calc( 0.5 * calc( 782px - 32px));
	}
}
```